core/src/main/scala/cats/data/NonEmptyMapImpl.scala

```diff
@@ -317,10 +317,19 @@ sealed abstract private[data] class NonEmptyMapInstances extends NonEmptyMapInst
   implicit def catsDataShowForNonEmptyMap[K: Show, A: Show]: Show[NonEmptyMap[K, A]] =
     Show.show[NonEmptyMap[K, A]](_.show)
 
-  implicit def catsDataBandForNonEmptyMap[K, A]: Band[NonEmptyMap[K, A]] =
+  @deprecated("Use catsDataSemigroupForNonEmptyMap", "2.5.0")
+  def catsDataBandForNonEmptyMap[K, A]: Band[NonEmptyMap[K, A]] =
```

Comment:
can we not move this to a lower priority? I think we could.

Reply:
I don't think we should. Having a surprising `Semigroup` instance (as we do right now) is error-prone. But having two instances (one of them surprising), and choosing between them implicitly based on whether or not `A` has a `Semigroup` instance in scope - that is downright evil, IMO.

Reply:
Also it wouldn't work, because `Band <: Semigroup` so if we put a `Band` instance in a lower-priority scope, they will become ambiguous (lower priority gives you -1 points, but being a subtype gives you +1 points, making the score even).

Reply:
Yeah. This was a bad suggestion on my part.